substrate/runtime/executive/Cargo.toml

```diff
@@ -17,6 +17,7 @@ substrate-runtime-system = { path = "../system", default_features = false }
 
 [dev-dependencies]
 substrate-primitives = { path = "../../primitives" }
+substrate-runtime-balances = { path = "../balances" }
```

Comment:
Should balances be added to the features.std array?

Reply:
no - it's just a dev dependency so it always has `std`